tests/scripts/all.sh

```diff
@@ -149,7 +150,7 @@ cleanup()
 {
     command make clean
 
-    find . -name yotta -prune -o -iname '*cmake*' -not -name CMakeLists.txt -exec rm -rf {} \+
+    find . -name yotta -prune -o -name .git -prune -o -iname '*cmake*' -not -name CMakeLists.txt -exec rm -rf {} \+
```

Comment:
Just in case other users use subversion or something, as well as having `.git` in the `find`, I think we need a comment.

Reply:
I think in the context of `all.sh` this is fine, as we're running git commands two lines below, so clearly we're assuming we're in a git repo already. Also, `all.sh` already has pretty strict environment requirements (assumes two version of OpenSSL and GnuTLS installed, etc.) which means it's only useful on developer or CI machines, so I think this should be acceptable.